install/resources/settings.xml

```diff
@@ -2452,15 +2452,6 @@ delete=Delete]]></optionscode>
 			<isdefault>1</isdefault>
 			<helpkey></helpkey>
 		</setting>
-		<setting name="allowaimfield">
-			<title>Allow AIM Screen Name To Usergroups</title>
-			<description><![CDATA[Select the usergroups which should be allowed to use the AIM Screen Name contact field.]]></description>
-			<disporder>2</disporder>
```

Comment:
Since you are removing this setting you need to downgrade the `disporder` for the rest of the settings. Thus, `allowyahoofield` `disporder` value would be `2` now an so on.

Reply:
Ok will do. But is there any impact?

Reply:
Not really. The display order can be assigned within the install code. I suggested removing this way back but never got to do so myself.

Until we remove it we should try to be consistent.

Reply:
Done.